link/src/main/java/com/stripe/android/link/LinkActivityViewModel.kt

```diff
@@ -0,0 +1,31 @@
+package com.stripe.android.link
+
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.ViewModelProvider
+import androidx.navigation.NavHostController
+
+internal class LinkActivityViewModel : ViewModel() {
+    var navController: NavHostController? = null
+    var dismissWithResult: ((LinkActivityResult) -> Unit)? = null
+
+    fun handleViewAction(action: LinkAction) {
+        when (action) {
+            LinkAction.BackPressed -> handleBackPressed()
+        }
+    }
+
+    private fun handleBackPressed() {
+        dismissWithResult?.invoke(LinkActivityResult.Canceled())
```

Comment:
I think this should be called something else. handleBackPressed sounds like it's for navigating back between screens, and then maybe exiting the whole activity if the back stack is empty. But the implementation looks like it just cancels.

Reply:
I was going to leave this for when I start implementing screens, but it makes sense to just add the back navigation now